config/routes.rb

```diff
@@ -5,5 +5,5 @@
   # root "articles#index"
```

Comment:
[OPTIONAL] It would be great idea if you make users#index as a root

Reply:
Thanks. Updated